tests/e2e/support/objects/app-files/share/index.ts

```diff
@@ -66,8 +69,16 @@ export class Share {
     return await hasPermissionToShare({ page: this.#page, resource })
   }
 
-  async copyQuickLink(args: Omit<copyLinkArgs, 'page'>): Promise<void> {
-    await copyQuickLink({ ...args, page: this.#page })
+  async copyAndGetQuickLinkUrl(args: Omit<copyLinkArgs, 'page'>): Promise<void> {
+    this.#linksEnvironment = new LinksEnvironment()
```

Comment:
need to preserve it?
```suggestion
    const linksEnvironment = new LinksEnvironment()
```